docs/code-samples/README.md

```diff
@@ -14,7 +14,7 @@ running the `daml new quickstart-finance` command.
 
 The [getting-started](getting-started/) code and the other code samples are fully replicated
 inside the
-[daml repository](https://github.com/digital-asset/daml/blob/main/templates/BUILD.bazel#L113).
+[daml repository](https://github.com/digital-asset/daml/blob/main/sdk/templates/BUILD.bazel).
```

Comment:
Also removing the #L113 line reference. I think it could get outdated quite quickly. For example, in the 2.9 release branch it is around line 117 but in main around line 67.